app/src/features/surveys/animals/profile/captures/components/capture-card-details/components/CaptureDetails.tsx

```diff
@@ -18,7 +18,7 @@ interface ICaptureDetailsProps {
 export const CaptureDetails = (props: ICaptureDetailsProps) => {
   const { capture } = props;
 
-  const captureTimestamp = capture.capture_timestamp;
+  const captureTimestamp = capture.capture_date;
```

Comment:
nit: rename `captureTimestamp` to `captureDate` (just for consistency)

Reply:
fixed